airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/copy/CopyConsumerFactory.java

```diff
@@ -77,7 +77,7 @@ private static <T> Map<AirbyteStreamNameNamespacePair, StreamCopier> createWrite
     final String stagingFolder = UUID.randomUUID().toString();
     for (final var configuredStream : catalog.getStreams()) {
       final var stream = configuredStream.getStream();
-      final var pair = AirbyteStreamNameNamespacePair.fromAirbyteSteam(stream);
+      final var pair = AirbyteStreamNameNamespacePair.fromAirbyteStream(stream);
```

Comment:
non-blocking: since we're being more explicit with `AirbyteStreamNameNamespacePair`, should this `var` be `AirbyteStreamNameNamespacePair` instead?